app.py

```diff
@@ -15,6 +16,10 @@
 # whatever the makefile says...
 app.config.from_envvar('ENV_CONFIG_FILE')
 
+# setup logging
+fileConfig('logging.config')
```

Comment:
let's put logging into the `config` folder alongside the ENV var ones. there's gunna be other .config files and i don't want them littering up the root dir